.github/workflows/python-publish.yml

```diff
@@ -0,0 +1,32 @@
+# This workflows will upload a Python Package using Twine when a release is created
+# For more information see: https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#publishing-to-package-registries
+
+name: Upload Python Package
+
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  deploy:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install setuptools wheel twine
+    - name: Build and publish
+      env:
+        TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+        TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+      run: |
+        python setup.py sdist bdist_wheel
+        twine upload dist/*
```

Comment:
This line (`twine upload dist/*`) might work here, since it has just set up a fresh environment, and presumably there will only be one file in `dist/` after executing the previous line.

But when I do this locally, it doesn't work, because files build up in the `dist/` subdirectory, e.g.,

scadnano-0.9.6.tar.gz
scadnano-0.9.7.tar.gz
scadnano-0.9.8.tar.gz

PyPI only lets you upload a file with a given name/version once, because there's no concept of updating a given version. If you want to change anything, you always have to create a new version.

Reply:
Let me try on my local fork.

Reply:
The workflow works fine for uploading future ones: https://pypi.org/project/scadnano-copy/0.9.13/#history

Workflow: https://github.com/UnHumbleBen/scadnano-python-package-1/runs/863738183?check_suite_focus=true